nixos/modules/services/games/factorio.nix

```diff
@@ -120,6 +201,7 @@ in
           "--config=${cfg.configFile}"
           "--port=${toString cfg.port}"
           "--start-server=${mkSavePath cfg.saveName}"
+          "--server-settings=${serverSettingsFile}"
           (optionalString (cfg.mods != []) "--mod-directory=${modDir}")
           (optionalString (cfg.autosave-interval != null) "--autosave-interval ${toString cfg.autosave-interval}")
```

Comment:
Oh! I meant to take it off the command line.. will do that now. Thanks for the catch.